pallets/nfts/src/lib.rs

```diff
@@ -236,6 +253,13 @@ impl<T: Config> NFTs for Module<T> {
     fn sealed(id: Self::NFTId) -> bool {
         Data::<T>::get(id).sealed
     }
+
+    fn burn(id: Self::NFTId) -> DispatchResult {
```

Comment:
@ETeissonniere  Quick question, should this function contain any kind of checks? Like what should happen if this is called with an inexistent NFT ID?

Reply:
This should be fine because we do assume this function is privileged (internal to the runtime). However if you do call this with a non existing ID it won't fail and will continue gracefully

Reply:
Makes sense, thanx!